core/arch/arm/kernel/boot.c

```diff
@@ -914,7 +914,7 @@ static uint64_t get_dt_val_and_advance(const void *data, size_t *offs,
 	return rv;
 }
 
-static int get_memory_all(void *fdt, struct core_mmu_phys_mem *mem)
+static int get_nsec_memory_helper(void *fdt, struct core_mmu_phys_mem *mem)
```

Comment:
minor: I suggest to rather rename it to `get_nsec_shm_helper()` since this finds shm, not specifically nsec memory.

Reply:
`nsec_shm` sounds weird and slightly confusing with the other shm functions dealing with the static shm area IMO.
I propose adding a comment.

Reply:
fine. thanks.